src/generate_list.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
This looks like a straight up import of the upstream script (https://github.com/keylime/keylime/blob/master/scripts/create_allowlist.sh)

Could you put a comment in this file to be explicit about where it came from?

Reply:
Ah, I see there is a Makefile target that produces this...up to you if you still want to stamp it with it's original location